dist/angular-localForage.js

```diff
@@ -14,7 +14,11 @@
       return factory(angular, localforage);
     });
   } else if(typeof exports === 'object' || typeof global === 'object') {
-    module.exports = factory(angular, require('localforage')); // Node/Browserify
+    if(typeof module === 'undefined') {
+      global.module.exports = factory(angular, require('localforage')); // NW.js
+    } else {
+      modules.exports = factory(angular, require('localforage')); // Node/Browserify
```

Comment:
Pushed my changes with the typo fix

Am I being dumb and missing something? (I am rather new to Nodejs overall) 
Here is what happens when I try to require from the root of my repo:

>brad:angular-localForage$node
>\> var localForage = require('.')
ReferenceError: window is not defined
    at /home/brad/Documents/GitHub/angular-localForage/dist/angular-localForage.js:11:44
    at Object.<anonymous> (/home/brad/Documents/GitHub/angular-localForage/dist/angular-localForage.js:25:3)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at repl:1:19
> \>

also tried
`var localForage = require('angular-localforage')`
same error message as above.